independent-publisher-connectors/WeatherForecast/apiDefinition.swagger.json

```diff
@@ -0,0 +1,229 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "weatherforecast",
+    "description": "The weather forecast connector helps one to get  weather forecast depending on the city of their choice",
+    "version": "1.0"
+  },
+  "host": "api.openweathermap.org",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/data/2.5/weather": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "coord": {
+                  "type": "object",
+                  "properties": {
+                    "lon": {
+                      "type": "number",
+                      "format": "float",
+                      "description": "lon"
+                    },
+                    "lat": {
+                      "type": "number",
+                      "format": "float",
+                      "description": "lat"
+                    }
+                  },
+                  "description": "coord"
+                },
+                "weather": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "id"
+                      },
+                      "main": {
+                        "type": "string",
+                        "description": "main"
+                      },
+                      "description": {
+                        "type": "string",
+                        "description": "description"
+                      },
+                      "icon": {
+                        "type": "string",
+                        "description": "icon"
+                      }
+                    }
+                  },
+                  "description": "weather"
+                },
+                "base": {
+                  "type": "string",
+                  "description": "base"
+                },
+                "main": {
+                  "type": "object",
+                  "properties": {
+                    "temp": {
+                      "type": "number",
+                      "format": "float",
+                      "description": "temp"
+                    },
+                    "feels_like": {
+                      "type": "number",
+                      "format": "float",
+                      "description": "feels_like"
```

Comment:
@Haimantika All the description values should be free from grammatical and spell errors. Please provide valid description values in the English language.

Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)